### PR TITLE
Updated project to use cerner-icons-v1.0.1

### PR DESCRIPTION
**Pull request by bjankord**
### Summary
Summarize the contents of the code changes in your pull request. Tag any open issues you believe to be resolved by this pull request.

This PR consumes cerner-one-icons v1.0.1 which contains a bugfix for duplicated icon names.
Previously there were two `next` icons and two `prev` icons. Now, the next and prev icons are uniquely named.

screen shot 2017-01-11 at 4 47 43 pm
screen shot 2017-01-11 at 4 47 53 pm
screen shot 2017-01-11 at 4 48 07 pm
screen shot 2017-01-11 at 4 48 21 pm

### CODE REVIEW

File: dev/scripts/generate-icon-sprite.js [hunk 1]
```diff
@@ -2,11 +2,6 @@ const fs = require('fs');
 const shell = require('shelljs');
 
 /**
- * NPM install cerner-one-icons when building sprite since it is a peer-dependency
```
Comment (bjankord) [inline dev/scripts/generate-icon-sprite.js]: Removing this as we can't guarantee master will always be the version we want to pull in, this versioning is noted in the package.json

File: src/cerner-one-icon-mapping.csv [hunk 2]
```diff
@@ -68,6 +68,7 @@ information_inverse,src/icons/0038_italic-i_i_heavyCircleWhite.svg
 italic-i,src/icons/0038_italic-i_a.svg
```
Comment (bjankord) [inline src/cerner-one-icon-mapping.csv]: This file is generated from cerner-one-icons repo.

File: tests/visual/terra-icon.html [hunk 3]
```diff
@@ -1642,6 +1642,27 @@
       <div style="font-size: 2em;">
```
Comment (bjankord) [inline tests/visual/terra-icon.html]: This file is generated via `npm run build-icons` based on CSV from cerner-one-icons repo.

**Comment by Matt-Butler** [comment]
+1

**Comment by mjhenkes** [comment]
+1

**Pull request merged**